code/modules/client/preference_setup/loadout/items/accessories.dm

```diff
@@ -335,6 +335,17 @@
 	scarfs["zebra scarf"] = /obj/item/clothing/accessory/scarf/zebra
 	gear_tweaks += new /datum/gear_tweak/path(scarfs)
 
+/datum/gear/accessory/shawl
+	display_name = "colorable shawl"
+	path = /obj/item/clothing/accessory/shawl
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/shawl/New()
+	..()
+	var/list/shawls = list()
+	shawls["cozy shawl"] = /obj/item/clothing/accessory/shawl
+	gear_tweaks += new /datum/gear_tweak/path(shawls)
```

Comment:
code is fine otherwise, but you don't need the `/New()` part if there's just one thing in the list (it's used for giving the dropdown to pick from different types of the same item), so this can be removed until more get added

Reply:
good to know, i just copy pasted the text from the scarf :D

Reply:
fastest reply in history - npnp!